src/commands/bootstrap/index.ts

```diff
@@ -0,0 +1,685 @@
+
+      // Content for the .env.local file
+      const content = `DATABASE_URL=${connectionString}
+AUTH_SECRET=${authSecret}`;
+
+      // Write the content to the .env.local file
+      writeFileSync(`${appName}/.env.local`, content, 'utf8');
+      writeFileSync(`${appName}/.dev.vars`, content, 'utf8'); // cloudflare
+      environmentVariables.push({
+        key: 'DATABASE_URL',
+        value: connectionString,
+        kind: 'build',
+      });
+      environmentVariables.push({
+        key: 'DATABASE_URL',
+        value: connectionString,
+        kind: 'runtime',
+      });
+      environmentVariables.push({
+        key: 'AUTH_SECRET',
+        value: authSecret,
+        kind: 'build',
+      });
+      environmentVariables.push({
+        key: 'AUTH_SECRET',
+        value: authSecret,
+        kind: 'runtime',
+      });
+    } else {
+      // Content for the .env.local file
+      const content = `DATABASE_URL=${connectionString}`;
+
+      // Write the content to the .env.local file
+      writeFileSync(`${appName}/.env.local`, content, 'utf8');
+      writeFileSync(`${appName}/.dev.vars`, content, 'utf8'); // cloudflare
+      environmentVariables.push({
+        key: 'DATABASE_URL',
+        value: connectionString,
+        kind: 'build',
+      });
+      environmentVariables.push({
+        key: 'DATABASE_URL',
+        value: connectionString,
+        kind: 'runtime',
+      });
+    }
+
+    out.text(
+      `Created a Next.js project in ${picocolors.blue(
+        appName,
+      )}.\n\nYou can now run ${picocolors.blue(
+        `cd ${appName} && npm run dev`,
+      )}`,
+    );
+  }
+
+  if (finalOptions.orm === 'drizzle') {
+    try {
+      execSync('npm run db:generate -- --name init_db', {
+        cwd: appName,
+        stdio: 'inherit',
+      });
+    } catch (error) {
+      throw new Error(`Generating the database schema failed: ${error}.`);
+    }
+
+    // If the user doesn't specify Auth.js, there is no schema to be applied.
+    if (finalOptions.auth === 'auth.js') {
+      try {
+        execSync('npm run db:migrate', { cwd: appName, stdio: 'inherit' });
+      } catch (error) {
+        throw new Error(`Applying the schema failed: ${error}.`);
+      }
+    }
+
+    out.text(`Database schema generated and applied.\n`);
+  }
+
+  const { deployment } = await prompts({
+    onState: onPromptState,
+    type: 'select',
+    name: 'deployment',
+    message: `Where would you like to deploy?`,
+    choices: [
+      {
+        title: 'Vercel',
+        value: 'vercel',
+        description: 'We will install the Vercel CLI globally.',
+      },
+      {
+        title: 'Cloudflare',
+        value: 'cloudflare',
+        description: 'We will install the Wrangler CLI globally.',
+
+        // If Yarn is chosen, Cloudflare doesn't work because I can't get `npx
+        // @cloudflare/next-on-pages` to work with the Yarn CLI.
+        disabled: finalOptions.packageManager === 'yarn',
```

Comment:
Let's reflect this downside in option title

Reply:
Would also love to understand this!

Latest yarn has `yarn dlx` which should be equivalent to `npx` I think.